examples/simple/thrift/client.py

```diff
@@ -1,9 +1,11 @@
 import json
```

Comment:
@abhinav are we sure that it's a requirement to have the license at the top of each file? I feel like with that and long doc blocks, it's really hard to get to the actual implementation :)